gatsby/content/docs/free-small-matrix-server.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: 'Free Small Matrix Server'
+author: Paul Tötterman <paul.totterman@iki.fi>
+layout: post
+section: guides
+license: CC-BY-NC-SA ?
+slug: free-small-matrix-server
+---
+
+*Canonical version of this article at <https://gitlab.com/ptman/matrix-docs/tree/master/free-matrix-server>*
+
+# Free Small Matrix Server
+
+Ingredients:
+
+- Domain
+- Server
+- Software
+- Configuration
+
+## Free domain name
+
+Get one from [Freenom] for free or any other place you wish. Freenom only gives
+free domains for a year. A Matrix server cannot be migrated (yet?) from one
+domain to another, so you might want to get a longer-lived one.
+
+- https://tld-list.com/ - find cheap domains, look at renewal fees also
+- https://www.namecheap.com/promos/amazing98s/ - cheap first year
+
+You also need to specify subdomains (which is why most dynamic dns services
```

Comment:
Which subdomains?

Edit: Oh, riot./matrix. ?

Reply:
so, it's ok as they're mentioned later?